### PR TITLE
Completed Diagnostic questions 03/24

### PR DESCRIPTION
**Pull request by Ryaong792**
questions related to GIT HUB pull/branch etc answered.

### CODE REVIEW

**Comment by Ryaong792** [comment]
completed

File: diagnostic.md [hunk 1]
```diff
@@ -7,40 +7,40 @@ What command do we use to do that? Use the **correct** URL from your fork of
...
-<!-- Remove this comment and place your answer here. -->
+git checkout -b response
 ```
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git log
```
Comment (jordanallain) [inline diagnostic.md:29]: Close we were looking for `git status`

`git log` will show you your past commits but not the changes from the working directory

File: diagnostic.md [hunk 2]
```diff
@@ -7,40 +7,40 @@ What command do we use to do that? Use the **correct** URL from your fork of
...
 6.Once `diagnostic.md` is staged, we have to make a commit by `git commit`.
 What are the two formatting items you **need** to make up your commit message?
 
-<!-- Remove this comment and place your answer here. -->
+1. good title    2. explain what change your made/do
 
 -7.Should you ever edit published history?
 
- <!-- Remove this comment and place your answer here. -->
+ Not really
```
Comment (jordanallain) [inline diagnostic.md:46]: An emphatic NO! would also be acceptable.

**Comment by jordanallain** [comment]
7/9

Great work Ryan! You almost nailed all of them. We wanted two commits for this diagnostic but you only had one so I docked a point there. We were also looking for `git status` to question 4. `git log` will show you your commit history where `git status` will show you the changes in your working directory.

**Pull request closed without merging**